TestApp/BAL/BALClass.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TestApp.Model;
+
+namespace TestApp.BAL
+{
+    
+    public class BALClass
```

Comment:
whats the goal of this class? all the business logic its on the controller. Please move logic here

Reply:
Leave it blank because when create project from scratch i was thinking to make that  class in which i will implement the business logic and using context .. But after that i prefer repository , It make loose coupling